objax/zoo/resnet_v2.py

```diff
@@ -335,6 +356,98 @@ def __init__(self,
         super().__init__(in_channels=in_channels,
                          num_classes=num_classes,
                          blocks_per_group=(3, 24, 36, 3),
+                         group_strides=(2, 2, 2, 1),
                          bottleneck=True,
                          normalization_fn=normalization_fn,
                          activation_fn=activation_fn)
+
+
+def convert_bn_layer(keras_layer, objax_layer):
+    """Converts variables of batch normalization layer from Keras to Objax."""
+    shape = objax_layer.gamma.value.shape
+    objax_layer.gamma = objax.TrainVar(jn.array(keras_layer.__dict__['gamma'].numpy()).reshape(shape))
+    objax_layer.beta = objax.TrainVar(jn.array(keras_layer.__dict__['beta'].numpy()).reshape(shape))
+    objax_layer.running_mean = objax.StateVar(jn.array(keras_layer.__dict__['moving_mean'].numpy()).reshape(shape))
+    objax_layer.running_var = objax.StateVar(jn.array(keras_layer.__dict__['moving_variance'].numpy()).reshape(shape))
+
+
+def convert_conv_layer(keras_layer, objax_layer):
+    """Converts variables of convolution layer from Keras to Objax."""
+    objax_layer.w = objax.TrainVar(jn.array(keras_layer.__dict__['kernel'].numpy()))
+    if keras_layer.__dict__.get('bias') is not None:
+        bias_shape = (objax_layer.w.value.shape[-1], 1, 1)
+        objax_layer.b = objax.TrainVar(jn.array(keras_layer.__dict__['bias'].numpy()).reshape(bias_shape))
+    else:
+        objax_layer.b = None
+
+
+def convert_linear_layer(keras_layer, objax_layer):
```

Comment:
`convert_linear_layer` and `convert_conv_layer` are almost identical, except for the line which computes `bias_shape`.
However for `bias_shape` I think you can just use code `bias_shape = objax_layer.b.value.shape`.

So I think it's better to have one function, instead of two identical functions. You can call it `convert_conv_or_linear_layer`

Reply:
Done. However, `bias_shape = objax_layer.b.value.shape` didn't work because currently objax conv layers are all defined with `use_bias=False`, while some keras layers use biases.